### PR TITLE
fix the example scripts and isadjoint

### PR DESCRIPTION
**Pull request by ziyiyin97**
1. Fix operators in the examples which are deprecated
2. Fix `isadjoint` for real domain and complex range operators

### CODE REVIEW

**Comment by mloubout** [comment]
Examples are small so maybe add them to CI, GTG.

**Comment by ziyiyin97** [comment]
Also fix the `isadjoint` so it works for real domain and complex range (or vice versa). Let me know if this looks ok

File: .github/workflows/ci-example.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      - name: Setup julia
+        uses: julia-actions/setup-julia@v1
+        with:
+          version: ${{ matrix.version }}
+          arch: ${{ matrix.arch }}
+  
+      - name: Build JOLI
+        uses: julia-actions/julia-buildpkg@latest
+
+      - name: Run examples
```
Comment (mloubout) [inline .github/workflows/ci-example.yaml]: that's overkill for this small, just add that one step to the standard CI.

Comment (ziyiyin97) [reply .github/workflows/ci-example.yaml]: ok

File: .github/workflows/ci-joli.yaml [hunk 2]
```diff
@@ -63,3 +63,10 @@ jobs:
 
       - name: Run tests
         uses: julia-actions/julia-runtest@latest
+
+      - name: Run examples
+        working-directory: examples
+        run: |
+          julia -e 'using Pkg; Pkg.add("JOLI")'
+          julia -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd()))'
```
Comment (mloubout) [inline .github/workflows/ci-joli.yaml]: That's adding JOLI twice, once in `add` and once in `dev`. JOLi already instantiated an simple `julia --project` should be enough

**Pull request merged**